apps/server/src/modules/provisioning/strategy/sanis/sanis-response.mapper.ts

```diff
@@ -120,12 +124,16 @@ export class SanisResponseMapper {
 		}
 
 		let otherUsers: ExternalGroupUserDto[] | undefined;
-		if (group.sonstige_gruppenzugehoerige) {
-			otherUsers = group.sonstige_gruppenzugehoerige
-				.map((relation: SanisSonstigeGruppenzugehoerigeResponse): ExternalGroupUserDto | null =>
-					this.mapToExternalGroupUser(relation)
-				)
-				.filter((otherUser: ExternalGroupUserDto | null): otherUser is ExternalGroupUserDto => otherUser !== null);
+		if (this.provisioningFeatures.schulconnexOtherGroupusersEnabled) {
```

Comment:
Please simplify this if-if cascade.
`if (this.provisioningFeatures.schulconnexOtherGroupusersEnabled) {
    otherUsers = group.sonstige_gruppenzugehoerige
        ? group.sonstige_gruppenzugehoerige
            .map(relation => this.mapToExternalGroupUser(relation))
            .filter(otherUser => otherUser !== null) as ExternalGroupUserDto[]
        : [];
}`